pt/orm/retrieving-data-and-resultsets.rst

```diff
@@ -1,4 +1,4 @@
-Retornando dados e conjuntos de resultados
+Requisitando Dados e Conjuntos de Resultados
 ##########################################
```

Comment:
Headlines needs to be as long as the words, or our tooling won't build the docs.